src/site/content/en/blog/responsive-web-design-basics/index.md

```diff
@@ -0,0 +1,579 @@
+---
+title: Responsive Web Design Basics
+subhead: How to create sites which respond to the needs and capabilities of the device they are viewed on.
+description: |
+  How to create sites which respond to the needs and capabilities of the device they are viewed on.
+date: 2019-02-12
+updated: 2020-05-14
+authors:
+ - petelepage
+ - rachelandrew
+tags:
+ - blog
+ - css
+ - layout
+ - mobile
+ - ux
+draft: true
+---
+
+* [Set the viewport](#viewport)
+* [Size content to the viewport](#size-content)
+* [Use CSS media queries for responsiveness](#media-queries)
+* [How to choose breakpoints](#breakpoints)
+* [View media query breakpoints in Chrome DevTools](#devtools)
+
+The use of mobile devices to surf the web continues to grow at an astronomical pace, and these devices are often constrained by display size and require a different approach to how content is laid out on the screen.
+
+Responsive web design, originally defined by [Ethan Marcotte in A List Apart](http://alistapart.com/article/responsive-web-design/), responds to the needs of the users and the devices they're using. The layout changes based on the size and capabilities of the device. For example, on a phone users would see content shown in a single column view; a tablet might show the same content in two columns.
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="resize.webm" type="video/webm; codecs=vp8">
+    <source src="resize.mp4" type="video/mp4; codecs=h264">
+  </video>
+</figure>
+
+A multitude of different screen sizes exist across phones, "phablets," tablets, desktops, game consoles, TVs, and even wearables. Screen sizes are always changing, so it's important that your site can adapt to any screen size, today or in the future. In addition, devices have different features with which we interact with them. For example some of your visitors will be using a touchscreen. Modern responsive design considers all of these things to optimize the experience for everyone.
+
+
+## Set the viewport {: #viewport }
+
+Pages optimized for a variety of devices must include a meta viewport tag in the head of the document.
+A meta viewport tag gives the browser instructions on how to control the page's dimensions and scaling.
+
+To attempt to provide the best experience, mobile browsers render
+the page at a desktop screen width (usually about `980px`, though this varies
+across devices), and then try to make the content look better by increasing
+font sizes and scaling the content to fit the screen.
+This means that font sizes may appear inconsistent to users,
+who may have to double-tap or pinch-to-zoom
+in order to see and interact with the content.
+
+```html/4
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    …
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    …
+  </head>
+  …
+
+```
```

Comment:
I forgot to ask, what was your reasoning for why you would have disagreed to providing more context if we didn't have the code highlighting feature? Just want to have a record of it somewhere.

Reply:
Only that it is then less obvious which bit you are talking about, so if there is no way to highlight it's generally better imo to only show the thing you are talking about, if we can highlight then adding the extra context is indeed preferable.